elements/pl-matrix-input/pl-matrix-input.mustache

```diff
@@ -122,12 +122,18 @@
 
 {{#format}}
 <p>
-Your answer must be a matrix (i.e., a 2D array). It can be expressed in one of two ways:
+    Your answer should be a matrix (i.e., a 2D array), or a vector (a 1D array).
```

Comment:
I think that we should stick with the current wording for the help text, because we really are always entering a 2d array. It's just that we are accepting Matlab OR Python syntax, and `[1,2,3]` is a valid Matlab 2d array so we accept it. That is, we should undo all changes, but add one additional note that I'll write below.